source/content/guides/autopilot/05-troubleshoot.md

```diff
@@ -109,10 +113,259 @@ Alternatively, you can add the plugin or theme to the **Excluded Updates** list
 
 </Accordion>
 
-## Unexpected Error Preventing Autopilot From Taking Screenshots of Your Website
+## Extension Updates are Missing
+
+<Accordion title="Could not apply the updates because a plugin or theme was not found while attempting the update." id="missing-extension-updates" icon="info-sign">
+
+### Issue
+
+This error message is displayed when a WordPress plugin or theme is not found when updates were attempted.
+
+### Diagnosis
+
+To determine which plugin is causing the issue:
+
+1. Check the uncommitted changes for Autopilot Multidev. It is likely a plugin has been deleted and another plugin has been added.
+
+1. Compare the list of plugins between the Autopilot and Dev environments and determine if there are any discrepancies. Ensure no entries are missing. If there is variance between the lists, such as a plugin that has been omitted or not updated on one of the lists, the plugin will be disabled on the Autopilot branch.
+
+If you need assistance, [contact Support](/guides/support/contact-support).
+
+### Solution
+
+Remove the plugin or theme from the site if it is not being used, or revert the plugin or theme to the original name.
+
+Alternatively, you can add the plugin or theme to the **Excluded Updates** list in Autopilot settings.
+
+</Accordion>
+
+## Preventing Autopilot From Taking Screenshots of Your Website
 
 <Accordion title="There was an unexpected error preventing Autopilot from taking screenshots of your website." id="unexpected-screenshot-error" icon="info-sign">
 
 Pantheon's Autopilot engineers investigate each of these errors as they occur. Please [contact Support](/guides/support/contact-support) via chat or ticket.
 
 </Accordion>
+
+## Failed Extension Updates
+
+<Accordion title="Could not apply the upstream updates." id="extension-updates-failed" icon="info-sign">
+
+### Issue
+
+Applying upstream updates failed.
+
+### Diagnosis
+
+This is most likely due to a merge conflict when applying upstream updates to the sites. For Composer sites, this could be caused by a failed Composer build or dependency conflicts.
+
+### Solution
+
+Resolve conflicts to apply updates. If a merge conflict is preventing you from merging a Multidev environment, follow the steps in the documentation for [Resolving Conflicts from Multidevs](/git-resolve-merge-conflicts#resolve-content-conflicts) and learn how to [Compare Multidev Environments Locally](/multidev#compare-multidev-environments-locally).
```

Comment:
- The merge conflict is between the upstream and the site code, not between multidev branch & master which this documentation seems to point to. resolving conflicts might be a similar process, but the sources are different than currently described.


- This solution does not have any docs re: what to do if the issue is composer.

Reply:
Ah, actually upstream updates is below, this section should be about extension updates, not upstream updates.

Reply:
I see we have composer docs in the correct section below.